Lib/asynchat.py

```diff
@@ -48,6 +48,13 @@
 import asyncore
 from collections import deque
 
+from warnings import warn
```

Comment:
We can't backport warnings like this, only documentation.

Reply:
Thanks, dropped changes to .py files, and adjusted the NEWS entry to refer only to docs (and not deprecation warnings).

I have made the requested changes; please review again.